test/functionalTest/cases/4286_Update_format_for_notification_messages/Update_format_for_notification_messages.test

```diff
@@ -0,0 +1,203 @@
+echo "===================================================================================="
+payload='
+{
+  "subject": {
+    "entities": [
+      {
+        "id": "E1"
+      }
+    ],
+    "condition": {
+      "attrs": [ "A1" ]
+    }
+  },
+  "notification": {
+    "httpCustom": {
+      "url": "http://localhost:'$LISTENER_PORT'/notify"
+    },
+    "attrs": [ "A1", "sub2_with_simplifiedKeyValues" ],
+    "attrsFormat": "simplifiedKeyValues"
+  },
+  "expires": "2050-04-05T14:00:00.00Z"
+}'
+orionCurl --url /v2/subscriptions --payload "$payload"
+sub3=$(echo "$_responseHeaders" | grep Location | awk -F/ '{ print $4 }' | tr -d "\r\n")
+echo
+echo
+
+
+echo "03. Create entity E1/T1"
+echo "======================="
+payload='{
+  "id": "E1",
+  "type": "T1",
+  "A1": {
+    "value": 1,
+    "type": "Number"
+  }
+}'
+orionCurl --url /v2/entities --payload "${payload}"
+echo
+echo
+
+
+echo "04. Dump accumulator to see notifications"
+echo "========================================="
+accumulatorDump
+echo
+echo
+
+
+--REGEXPECT--
+01. Create NGSIv2 subscription for E1/A1 with attrsFormat set to simplifiedNormalized
+=====================================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+02 Create NGSIv2 subscription for E1/A1 with attrsFormat set to simplifiedKeyValues
+===================================================================================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/subscriptions/REGEX([0-9a-f]{24})
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+03. Create entity E1/T1
+=======================
+HTTP/1.1 201 Created
+Content-Length: 0
+Location: /v2/entities/E1?type=T1
+Fiware-Correlator: REGEX([0-9a-f\-]{36})
+Date: REGEX(.*)
+
+
+
+04. Dump accumulator to see notifications
+=========================================
+POST http://localhost:REGEX(\d+)/notify
+Fiware-Servicepath: /
+Content-Length: 72
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Ngsiv2-Attrsformat: simplifiedNormalized
+Host: localhost:REGEX(\d+)
+Accept: application/json
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36}; cbnotif=[12345])
+
+[
+    {
+        "A1": {
+            "metadata": {},
+            "type": "Number",
+            "value": 1
+        },
+        "id": "E1",
+        "type": "T1"
+    }
+]
+=======================================
+POST http://localhost:REGEX(\d+)/notify
+Fiware-Servicepath: /
+Content-Length: 72
+User-Agent: orion/REGEX(\d+\.\d+\.\d+.*)
+Ngsiv2-Attrsformat: simplifiedKeyValues
+Host: localhost:REGEX(\d+)
+Accept: application/json
+Content-Type: application/json; charset=utf-8
+Fiware-Correlator: REGEX([0-9a-f\-]{36}; cbnotif=[12345])
+
+[
+    {
+        "A1": {
+            "metadata": {},
+            "type": "Number",
+            "value": 1
+        },
```

Comment:
This is not working well... in simplifiedKeyValues it should be:

```suggestion
        "A1": 1
```

Reply:
Maybe it is due to [my above comment](https://github.com/telefonicaid/fiware-orion/pull/4298#discussion_r1147222316)

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4298/commits/1de26680286740a30ffeee899efe749412072f20